x-pack/plugins/fleet/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -410,44 +404,6 @@ export async function installTemplate({
     });
   }
 
-  // Datastream now throw an error if the aliases field is present so ensure that we remove that field.
-  const getTemplateRes = await retryTransientEsErrors(
-    () =>
-      esClient.indices.getIndexTemplate(
-        {
-          name: templateName,
-        },
-        {
-          ignore: [404],
-        }
-      ),
-    { logger }
-  );
-
-  const existingIndexTemplate = getTemplateRes?.index_templates?.[0];
-  if (
-    existingIndexTemplate &&
-    existingIndexTemplate.name === templateName &&
-    existingIndexTemplate?.index_template?.template?.aliases
-  ) {
-    const updateIndexTemplateParams = {
-      name: templateName,
-      body: {
-        ...existingIndexTemplate.index_template,
-        template: {
-          ...existingIndexTemplate.index_template.template,
-          // Remove the aliases field
-          aliases: undefined,
-        },
-      },
-    };
-
-    await retryTransientEsErrors(
-      () => esClient.indices.putIndexTemplate(updateIndexTemplateParams, { ignore: [404] }),
-      { logger }
-    );
-  }
-
```

Comment:
AFAICT this isn't necessary. Creating an index template without the `aliases` field at all seems to have the same behavior in my manual testing and this is already what the main create index template logic below does.

@nchaulet It looks like you wrote this originally, do you know if there's anything I'm missing here?

Reply:
Here's the original issue: https://github.com/elastic/kibana/issues/90984

It appears it was from an upgrade of the endpoint package that I can no longer reproduce, even using the old versions mentioned in the issue 0.16.2 -> 0.17.0

Reply:
I believe it could be reproduced if you upgraded from 7.10.0 to main, but this is not strictly supported since Fleet was not GA until 7.14.0. I think this is a safe thing to remove.